src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileServiceImpl.java

```diff
@@ -184,5 +200,101 @@ private PaymentFileLoad getCustomerProfileFromUnparsableFile(String incomingFile
         return paymentFile;
     }
 
+    /**
+     * Checks whether any of the batch load's payment groups reference inactive vendors,
+     * and sends warning emails appropriately if so.
+     * 
+     * @param paymentGroups The payment groups that were loaded.
+     * @param customer The customer's profile.
+     */
+    private void checkForInactiveVendors(List<PaymentGroup> paymentGroups, CustomerProfile customer) {
+        final int MESSAGE_START_SIZE = 300;
+        StringBuilder inactiveVendorsMessage = new StringBuilder(MESSAGE_START_SIZE);
+        VendorService vendorService = SpringContext.getBean(VendorService.class);
```

Comment:
Yes, that should be possible, just some Spring bean edits. I will make that change for consistency.